projects/storefrontlib/cms-components/product/carousel/product-carousel/product-carousel.component.html

```diff
@@ -11,17 +11,5 @@
 </cx-carousel>
 
 <ng-template #carouselItem let-item="item">
-  <a tabindex="0" [routerLink]="{ cxRoute: 'product', params: item } | cxUrl">
-    <cx-media
-      [container]="item.images?.PRIMARY"
-      format="product"
-      [alt]="item.name"
-    ></cx-media>
-    <h3>
-      {{ item.name }}
-    </h3>
-    <div class="price">
-      {{ item.price?.formattedValue }}
-    </div>
-  </a>
+  <cx-product-carousel-item [item]="item"></cx-product-carousel-item>
```

Comment:
We need some migration documentation for this slight dom change.
See docs/migration/5_0.md for some template change doc.